partner_contact_sale_info_propagation/tests/test_partner_contact_sale_info_propagation.py

```diff
@@ -54,9 +54,6 @@ def test_onchange_parent_id_with_values_false(self):
         partner_child = self.partner_model.create({"name": "Parent child"})
         partner_child.write({"parent_id": self.parent_company.id})
         onchange_result = partner_child.onchange_parent_id()
-        self.assertEqual(
```

Comment:
If you have to remove this, it means something is failing. Please also use `Form` for not having to manually call onchanges.

Reply:
Hey @pedrobaeza 

I used the Form as you advised me and changed the fields_view_get() method to get_view().